dense_vector/challenges/default.json

```diff
@@ -47,6 +52,34 @@
         "include-in-reporting": false
       }
     },
+    {
+      "parallel": {
+        "warmup-time-period": 10,
+        "completed-by": "index-update-concurrent-with-searches",
+        "tasks": [
+          {
+            "name": "index-update-concurrent-with-searches",
+            "operation": {
+              "operation-type": "bulk",
+              "bulk-size": 5000,
+              "ingest-percentage": 5,
+              "refresh" : "wait_for"
```

Comment:
I'm not sure if this matters, but curious why we don't use the default `"refresh": false`?

Reply:
Addressed in cd8ea4131654c67c0adf96a93f270d3ec371ed26.